content/blog/mobile-benchmarks.md

```diff
@@ -1,15 +1,16 @@
 ---
 title: Mobile Benchmarks
-attribution: Originally written by David Cao with additional content by Makarand and
-  others at Google.
-author: David Cao
-company: Google
-company-link: https://cloud.google.com
 date: 2016-07-26
-thumbnail: ../img/gcp-icon.png?raw=true
+author:
+  name: David Cao
+  position: Google
+  blurb: Originally written by David Cao with additional content by Makarand and others at Google
+thumbnail: /img/gcp-icon.png?raw=true
 ---
 
-As gRPC has become a better and faster RPC framework, we've consistently gotten the question, "How _much_ faster is gRPC?" We already have comprehensive server-side benchmarks, but we don't have mobile benchmarks. Benchmarking a client is a bit different than benchmarking a server. We care more about things such as latency and request size and less about things like queries per second (QPS) and number of concurrent threads. Thus we built an Android app in order to quantify these factors and provide solid numbers behind them. 
+As gRPC has become a better and faster RPC framework, we've consistently gotten the question, "How _much_ faster is gRPC?" We already have comprehensive server-side benchmarks, but we don't have mobile benchmarks. Benchmarking a client is a bit different than benchmarking a server. We care more about things such as latency and request size and less about things like queries per second (QPS) and number of concurrent threads. Thus we built an Android app in order to quantify these factors and provide solid numbers behind them.
+
+<!--more-->
```

Comment:
What's this for?

Reply:
Hugo has a built-in `.Summary` variable that contains a page's content up to this marker. That `.Summary` content is used on each blog post's "card" on the main blog page.

Reply:
Cool, thanks for the explanation.